spec/furniture/marketplace/collecting_payments_system_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+# @see https://github.com/zinc-collective/convene/issues/1576
+describe "Marketplace: Collecting Payments", type: :system do
+  # @see https://github.com/zinc-collective/convene/issues/1622
+  describe "via Stripe" do
+    it "Connecting a Vendor Stripe Account" do
+      space = create(:space, :with_entrance, :with_members)
+      marketplace = create(:marketplace, :with_stripe_utility, room: space.entrance)
+      sign_in(space.members.first, space)
+
+      visit polymorphic_path(marketplace.room.location)
+
+      within("##{dom_id(marketplace, :onboarding)}") do
+        click_on("Manage Marketplace")
+      end
```

Comment:
🤔 Here I'm wishing for a single test selector target, so there's no need to scope the dom selection.

Reply:
I tend to prefer scoping to dropping test ids; but I don't block folks who lean into test ids.